Classes/Controller/Cart/CartController.php

```diff
@@ -23,6 +24,8 @@
 
 class CartController extends ActionController
 {
+    protected $defaultViewObjectName = CartTemplateView::class;
```

Comment:
That looks a bit weird. I have seen that this is also done in \TYPO3\CMS\Extbase\Mvc\Controller\ActionController. Can't we override the variable in the __contruct method and assign it to the variable? Perhaps we could inject it to the controller so ones later can use an own implementation of CartTemplateView.

Reply:
I followed the official documentation: https://docs.typo3.org/m/typo3/book-extbasefluid/10.4/en-us/8-Fluid/2-using-different-output-formats.html#switching-php-class-of-view (or https://docs.typo3.org/c/typo3/cms-core/main/en-us/Changelog/10.0/Breaking-87511-RemoveViewFormatToObjectNameMapProperty.html#migration)
I do not know whether your ideas work as excepted. I must admit that it feels a bit over-engineered to me otherwise.